ern-runner-gen-android/src/AndroidRunnerGenerator.ts

```diff
@@ -19,12 +20,33 @@ export default class AndroidRunnerGenerator implements RunnerGenerator {
   }
 
   public async generate(config: RunnerGeneratorConfig): Promise<void> {
+    log.info(`generate config ${JSON.stringify(config)}`)
     let mustacheView: any = {}
     mustacheView = configureMustacheView(config, mustacheView)
-
     shell.cp('-R', path.join(runnerHullPath, '*'), config.outDir)
+    shell.mv(
+      path.join(
+        config.outDir,
+        'app',
+        'src',
+        'main',
+        'java',
+        'com',
+        'walmartlabs',
+        'ern',
+        'miniapp'
+      ),
+      path.join(
+        config.outDir,
+        'app',
+        'src',
+        'main',
+        'java',
+        config.extra.androidConfig.packageFilePath
+      )
+    )
```

Comment:
**minor** 
You can use `path.join` with the full path, i.e `path.join(config.outDir, 'app/src/main/java/com/walnartlabs/ern/miniapp')`. `path.join` will normalize the resulting path based on the platform. That's mostly my bad, used to do this very verbose way for most of the paths for no valid reason, but now doing it the shorter way instead.